backend/migrations/versions/3ab38446596e_create_user_model.py

```diff
@@ -0,0 +1,36 @@
+"""Create user model
```

Comment:
Putting this comment here for lack of a better place: Would it be useful (for our future selves) to add a `./backend/README.md` and use that as a place to document how devs can autogenerate / run migrations?